source/_integrations/vicare.markdown

```diff
@@ -118,6 +118,15 @@ Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode]
 | `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
 | `vicare_mode` | no | New value of ViCare mode. For supported values, see the `vicare_modes` attribute of the climate {% term entity %}. |
 
+### Action `vicare.set_dhw_circulation_pump_schedule`
+
+Set the schedule domestic hot water circulation pump as defined by Viessmann 
+
+| Data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
```

Comment:
```suggestion
| `entity_id` | yes | String or list of strings that point at `entity_id`s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
```